Sprint-3/2-practice-tdd/get-ordinal-number.test.js

```diff
@@ -8,6 +8,63 @@ const getOrdinalNumber = require("./get-ordinal-number");
 // When the number is 1,
 // Then the function should return "1st"
 
+test("should return NaN for 1.5", () => {
+  expect(getOrdinalNumber(1.5)).toEqual(NaN);
+});
+
+test("should return NaN for 0", () => {
```

Comment:
This is not critical but it doesn't really make sense. 0 is a number and returning NaN for it is a bit confusing. Same for 1.5. You might want to return something else if you don't support those numbers.